terraform/performance/main.tf

```diff
@@ -89,30 +89,24 @@ resource "null_resource" "integration_test" {
     destination = "${local.instance_temp_directory}/${local.cloudwatch_agent_config}"
   }
 
+  # Install agent binaries
   provisioner "remote-exec" {
     inline = [
-      "echo sha ${var.cwa_github_sha}",
       "cloud-init status --wait",
-      "echo clone and install agent",
-      "git clone --branch ${var.github_test_repo_branch} ${var.github_test_repo}",
-      "cd amazon-cloudwatch-agent-test",
-      "aws s3 cp s3://${var.s3_bucket}/integration-test/binary/${var.cwa_github_sha}/linux/${var.arc}/${var.binary_name} .",
-      "export PATH=$PATH:/snap/bin:/usr/local/go/bin",
-      var.install_agent,
-      "go run ./validator/main.go --validator-config=${local.instance_temp_directory}/${local.final_validator_config} --preparation-mode=true",
+      "aws s3 cp s3://${var.s3_bucket}/integration-test/binary/${var.cwa_github_sha}/linux/${var.arc}/amazon-cloudwatch-agent.rpm .",
+      "sudo rpm -U ./amazon-cloudwatch-agent.rpm"
     ]
   }
 
-
-  #Run sanity check and integration test
+  #Prepare the requirement before validation and validate the metrics/logs/traces
   provisioner "remote-exec" {
     inline = [
       "export AWS_REGION=${var.region}",
-      "export PATH=$PATH:/snap/bin:/usr/local/go/bin",
```

Comment:
Does this not require the go path to be exported? I remember of some os we have to do this or the tests will nto execute.

Reply:
Nope its only a requirement for Ubuntu. Not Linux but we won't support other similar Linux distribution